multicast/__main__.py

```diff
@@ -17,20 +17,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__all__ = ["""__module__""", """__prog__""", """__prolog__""", """__epilog__""", """__doc__""", "NoOp", """SendMCast""", """joinMCast""", """dumpUsage""", """TASK_OPTIONS""", """__checkToolArgs""", """buildArgs""", """main""", """parseArgs""", """useTool"""]
+__all__ = [
+	"""__module__""", """__prog__""", """__prolog__""", """__epilog__""",
```

Comment:
Fix #12 with:
```suggestion
	"""__module__""", """__proc__""", """__prolog__""", """__epilog__""",
```
missing variable should be `__proc__`